test/suite/test_tiered02.py

```diff
@@ -48,9 +48,11 @@ def conn_config(self):
           'bucket_prefix=%s,' % self.bucket_prefix + \
           'name=%s),tiered_manager=(wait=0)' % self.extension_name
 
-    # Load the local store extension, but skip the test if it is missing.
+    # Load the local store extension. 
     def conn_extensions(self, extlist):
-        extlist.skip_if_missing = True
+        # Windows doesn't support dynamically loading extension libraries.
```

Comment:
This sentence is correct as-is. But when I first read it I expected slightly different wording. Ignore this comment if you want. But either of these are what I expected to read: (if you change it, please change it in all the files)

Windows doesn't support dynamically loaded extension libraries.
Windows doesn't support loading extension libraries dynamically.

Reply:
Done.  (Using the first option)